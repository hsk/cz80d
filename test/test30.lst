01   0000 30 10         JR     NC,0x12
02   0002 31 34 12      LD     SP,0x1234
03   0005 32 34 12      LD     (0x1234),A
04   0008 33            INC    SP
05   0009 34            INC    (HL)
06   000A 35            DEC    (HL)
07   000B 36 12         LD     (HL),0x12
08   000D 37            SCF
09   000E 38 02         JR     C,0x12
10   0010 39            ADD    HL,SP
11   0011 3A 34 12      LD    A,(0x1234)
12   0014 3B            DEC    SP
13   0015 3C            INC    A
14   0016 3D            DEC    A
15   0017 3E 12         LD     A,0x12
16   0019 3F            CCF
