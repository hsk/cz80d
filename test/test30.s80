  JR     NC,0x12
  LD     SP,0x1234
  LD     (0x1234),A
  INC    SP
  INC    (HL)
  DEC    (HL)
  LD     (HL),0x12
  SCF
  JR     C,0x12
  ADD    HL,SP
  LD    A,(0x1234)
  DEC    SP
  INC    A
  DEC    A
  LD     A,0x12
  CCF
