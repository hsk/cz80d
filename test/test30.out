014243456789:4;<=>?